<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevator System Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="passengerSetup">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Passenger Setup</string>
    </property>
    <widget class="QTextEdit" name="passengerSetupText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="simulationControl">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>141</width>
      <height>162</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation Control</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="safetyEvents">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>220</y>
      <width>151</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Building Safety Events</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="fireButton">
       <property name="text">
        <string>Fire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="powerOutButton">
       <property name="text">
        <string>Power Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="elevator1Controls">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>125</width>
      <height>193</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator 1</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="elevator1Help">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator1DoorObstacle">
       <property name="text">
        <string>Door Obstacle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator1Overload">
       <property name="text">
        <string>Overload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator1OpenDoor">
       <property name="text">
        <string>Open Door</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator1CloseDoor">
       <property name="text">
        <string>Close Door</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="simulationStepLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation Step: 0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="logConsole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>Log Console</string>
    </property>
    <widget class="QTextEdit" name="logText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>541</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="elevator2Controls">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>125</width>
      <height>193</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator 2</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="elevator2Help">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator2DoorObstacle">
       <property name="text">
        <string>Door Obstacle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator2Overload">
       <property name="text">
        <string>Overload</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <widget class="QPushButton" name="elevator2OpenDoor">
       <property name="text">
        <string>Open Door</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elevator2CloseDoor">
       <property name="text">
        <string>Close Door</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGUI">
    <property name="title">
     <string>GUI</string>
    </property>
   </widget>
   <addaction name="menuGUI"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
